test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
* [ ] The `test.md` file should be ignored in your `.gitignore` file but I could see it in your repo. 
Please add it to your `.gitignore file`